rules/alerting.go

```diff
@@ -109,6 +109,12 @@ type AlertingRule struct {
 	// only after the restoration.
 	restored bool
 
+	// The health of the alerting rule.
+	health RuleHealth
+
+	// The last error seen by the alerting rule.
+	lastError error
+
```

Comment:
nit but since `health` and `lastError` are protected by the mutex, it would be better to have them after `mtx`.